test/functional/tests/stats/test_ioclass_stats.py

```diff
@@ -9,328 +9,666 @@
 
 from api.cas import casadm
 from api.cas import ioclass_config
-from api.cas.cache_config import CleaningPolicy, CacheMode, CacheLineSize
+from api.cas.cache_config import (
+    CleaningPolicy,
+    CacheMode,
+    CacheLineSize,
+    CacheModeTrait,
+    SeqCutOffPolicy,
+)
 from api.cas.casadm import StatsFilter
 from api.cas.cli_messages import (
     check_stderr_msg,
     get_stats_ioclass_id_not_configured,
-    get_stats_ioclass_id_out_of_range
+    get_stats_ioclass_id_out_of_range,
 )
+from api.cas.ioclass_config import IoClass
```

Comment:
cleanup imports:
`from api.cas import casadm` and `from api.cas.casadm import StatsFilter`
`from api.cas import ioclass_config` and `from api.cas.ioclass_config import IoClass` - furthermore later in code there is `IoClass = ioclass_config.IoClass`

Reply:
While technically this is correct, what I had in mind was importing necessary things from each module (it's like 3 references for each)